06_puppy/nickolee/storerInterface_test.go

```diff
@@ -0,0 +1,78 @@
+package puppystorer
```

Comment:
i think it's somewhat established to have go filenames all lowercase (same below).
storer_test.go should suffice - `er` usually indicates interface (e.g. Reader, Writer, Closer....)